todowebapp/src/main.jsx

```diff
@@ -0,0 +1,10 @@
+import { StrictMode } from 'react';
+import { createRoot } from 'react-dom/client';
+import App from './App.jsx';
+import './index.css';
+
+createRoot(document.getElementById('root')).render(
```

Comment:
```suggestion
createRoot(document.querySelector('main')).render(
```

This is safe: `main` elements are guaranteed to be unique.